.project-keeper.yml

```diff
@@ -0,0 +1,11 @@
+sources:
+  - type: maven
+    path: pom.xml
+    modules:
+      - integration_tests
+      - lombok
+      - udf_coverage
+      - jar_artifact
+excludes:
+  - "E-PK-CORE-18: Outdated content: '.github/workflows/ci-build.yml'"
+  - "E-PK-CORE-18: Outdated content: 'pk_generated_parent.pom'"
```

Comment:
Don't exclude the henerated parent!
It breaks the PK functionallity.

Reply:
I'll call you about this one. I needed to do it because I wanted to do some <excludes> instead of <includes> of integration tests.